projects/storefrontlib/src/cms-components/organization/order-approval/order-approval-list/order-approval-list.component.html

```diff
@@ -0,0 +1,175 @@
+<ng-container *ngIf="orderApprovals$ | async as orderApprovals">
+  <!-- BODY -->
+  <div class="cx-order-history-body">
+    <ng-container
+      *ngIf="orderApprovals.pagination.totalResults > 0; else noOrder"
+    >
+      <!-- Select Form and Pagination Top -->
+      <div class="cx-order-approval-sort top row">
+        <div
+          class="cx-order-approval-form-group form-group col-sm-12 col-md-4 col-lg-4"
+        >
+          <cx-sorting
+            [sortOptions]="orderApprovals.sorts"
+            [sortLabels]="getSortLabels() | async"
+            (sortListEvent)="changeSortCode($event)"
+            [selectedOption]="orderApprovals.pagination.sort"
+          ></cx-sorting>
+        </div>
+        <div class="cx-order-approval-pagination">
+          <cx-pagination
+            [pagination]="orderApprovals.pagination"
+            (viewPageEvent)="pageChange($event)"
+          ></cx-pagination>
+        </div>
+      </div>
+      <!-- TABLE -->
+      <table class="table cx-order-approval-table">
+        <thead class="cx-order-approval-thead-mobile">
+          <th scope="col">
+            {{ 'orderApprovalList.orderCode' | cxTranslate }}
+          </th>
+          <th scope="col">{{ 'orderApprovalList.POCode' | cxTranslate }}</th>
+          <th scope="col">{{ 'orderApprovalList.placedBy' | cxTranslate }}</th>
+          <th scope="col">{{ 'orderApprovalList.date' | cxTranslate }}</th>
+          <th scope="col">
+            {{ 'orderApprovalList.status' | cxTranslate }}
+          </th>
+          <th scope="col">{{ 'orderApprovalList.total' | cxTranslate }}</th>
+        </thead>
+        <tbody>
+          <tr *ngFor="let approval of orderApprovals.values">
```

Comment:
Indeed, since every field in the row is clickable, I didn't see the need to make the whole row clickable.  Moreover, there is no kind of visual feedback to let the user know that the white space in the row is clickable ( like the row being highlighted on hover and/or the cursor changes shape ).

However, if we favour consistency between the 2 tables, I can add it back though.  Let's discuss.

Reply:
After testing a bit more, I suppose this mechanic was added to serve the mobile view.  The table looks different win mobile view and perhaps we don't want the users to aim for the links.  

I added back the table row click.